online-shop/src/app/app.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+
+  title = 'online-shop';
+  readonly ROOT_URL = 'http://localhost:3000/products';
+  posts: any;
+  constructor(private http: HttpClient){}
+
+  getPosts2(){
```

Comment:
**suggestion:** what happened to `getPosts1`? :)) . On the other hand take care how you name your methods/variables. Check out: https://angular.io/guide/styleguide#style-vocabulary